src/Public/Redis.ps1

```diff
@@ -17,36 +17,48 @@ function Connect-Redis
     param (
         [Parameter(Mandatory=$true)]
         [string]
-        $ConnectionString
+        $ConnectionString,
+
+        [Parameter()]
+        $ConnectionName
     )
 
     # first, disconnect any existing connection
-    Disconnect-Redis
+    Disconnect-Redis -ConnectionName $ConnectionName
+
+    if ($null -eq $ConnectionName)
```

Comment:
Might be better as `[string]::IsNullOrWhiteSpace($ConnectionName)`

Reply:
Done